linker.py

```diff
@@ -0,0 +1,103 @@
+import itertools
+
+import numpy as np
+import pandas as pd
+from sklearn.cluster import KMeans
+
+
+RANDOM_SEED = 0
```

Comment:
ah man, we gotta get geeky here. How about `46` for the 46 chromosomes we all have and the cause of all genetic diseases? :)

Reply:
Haha, there is a reason why I chose 0: I was testing a sample and the correct result was 4. With 0, the result was wrong, it gave 6, but basically with any other seed I got 4.

I thought changing the 0 would be considered cheating, then :D.